src/Osma.Mobile.App/Events/ApplicationEvent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Osma.Mobile.App.Events
+{
+    namespace Osma.Mobile.App.Events
+    {
+        public enum ApplicationEventType
+        {
+            ConnectionsUpdated
+        }
+
+        public class ApplicationEvent
+        {
+            public ApplicationEventType EventType { get; set; }
```

Comment:
Is it useful to duplicate the name `Event` here? The item is an event, could we rename this from `EventType` to `Type`
```suggestion
            public ApplicationEventType Type { get; set; }
```